tests/test_profile.py

```diff
@@ -155,12 +155,13 @@ def test_profile_extra_fields(client: FlaskClient, app: Flask) -> None:
     assert b"Signal username" in response.data
     assert b"singleusername.666" in response.data
     assert b"Arbitrary Link" in response.data
-    # URLs should turn into links
-    assert (
-        '<a href="https://scidsg.org/" target="_blank" rel="noopener noreferrer"'
-        "\n                  >https://scidsg.org/</a\n                >" in response.text
-    )
+
+    # URLs should turn into links - check that critical parts of the link are present
```

Comment:
The problem with how this is checking for specific parts of the link being present is that if any link on the page includes `target="_blank"` then this test will pass, even if this specific link doesn't have that. Instead you can use bs4 to find this specific link, and then make sure it has the properties you want it to have.